docs/api-reference/carto/overview.md

```diff
@@ -0,0 +1,55 @@
+# @deck.gl/carto
+
+Deck.gl is the prefered and official solution to create modern Location Intelligence Webapps with [CARTO platform](https://carto.com/)
+
+<img src="https://raw.githubusercontent.com/CartoDB/viz-doc/master/deck.gl/img/osm_buildings.jpg" />
+
+
+It integrates with [CARTO Maps API](https://carto.com/developers/maps-api/reference/) to:
+
+* Provide a geospatial backend storage for your geospatial data.
+* Visualize large datasets which don't fit in browser memory.
+* SQL spatial interface to work with your data.
+
+
+## Install package
+
+```bash
+npm install deck.gl
```

Comment:
For this to work the new module must be added to `modules/main`'s package.json

Reply:
Since we're not going to include in the bundle I think it's clearer to keep it outside the main module. Let me know otherwise.

Reply:
deck.gl can include all submodules as dependency. This makes installation easier and has no direct impact on bundle size unless the user references a specific API.

Reply:
Done!